src/client/binary_client.cpp

```diff
@@ -375,7 +375,7 @@ namespace
       request.MethodsToCall = methodsToCall;
       const CallResponse response = Send<CallResponse>(request);
       if (Debug) {std::cout << "binary_clinent | Call <--" << std::endl;}
-      return response.Results;
+      return response.Parameters.Results;
```

Comment:
seems to be a bug here, that python script was written very quickly. the
list is also in the generate_model.py file. Can you try to fix it? you need
to find which one is currently used and move it to generate_protocol.py. It
should be empty in generate_model.py

On Sun, 31 May 2015 at 13:52 Denis Štogl <notifications@github.com> wrote:

> In src/client/binary_client.cpp
> <https://github.com/FreeOpcUa/freeopcua/pull/143#discussion_r31389462>:
>
> > @@ -375,7 +375,7 @@ namespace
> >        request.MethodsToCall = methodsToCall;
> >        const CallResponse response = Send<CallResponse>(request);
> >        if (Debug) {std::cout << "binary_clinent | Call <--" << std::endl;}
> > -      return response.Results;
> > +      return response.Parameters.Results;
>
> That doesn't work since the list is not used anywhere, at least I can not
> find ti....
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/FreeOpcUa/freeopcua/pull/143/files#r31389462>.
>